rero_ils/modules/files/cli.py

```diff
@@ -100,18 +111,18 @@ def create_files(number):
     doc_pids = list(Document.get_all_pids())
     lib_pids = list(Library.get_all_pids())
     shuffle(doc_pids)
```

Comment:
Why we need `shuffle` and why we have `collections = ["col1", "col2", "col3"]`?
Maybe use:

```python
@click.option('-c', '--collections', multiple=True, default=["col1", "col2", "col3"])
def create_files(number, collections):
```